src/scripts/components/shapes.es6

```diff
@@ -24,7 +24,7 @@ module.exports = class Shapes extends Component {
 
 	injectSprite() {
 		$.get(this.data.url, (response, status) => {
-			if (status == 'success') {
+			if (status === 'success') {
```

Comment:
JS way je neřešit typy, dokud to nepotřebuješ. Já naopak konverzi vždy vítám, šetří mi práci. A tam, kde potřebuji to explicitně stejně převádím na potřebný typ.